extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/sources/generated/GeoJsonSourceTest.kt

```diff
@@ -81,7 +83,7 @@ class GeoJsonSourceTest : BaseStyleTest() {
       url(TEST_URI)
     }
     setupSource(testSource)
-    assertEquals(TEST_URI, testSource.data)
+    Handler(Looper.getMainLooper()).postDelayed({ assertEquals(TEST_URI, testSource.data) }, LATCH_MAX_TIME_MS)
```

Comment:
The test is not gonna wait for the `assertEquals` to run before moving to the next test. 
I just tested by calling `Assert.fail()` inside the runnable and the test doesn't fail.

Reply:
I think moving this test to the `extension-style` module would help with the testing because we would be able to access the `workerThread` to monitor if setting the data has been done.

Reply:
> The test is not gonna wait for the assertEquals to run before moving to the next test.
> I just tested by calling Assert.fail() inside the runnable and the test doesn't fail.

Yes, fixing that. need to apply latch to wait till handler completes

Reply:
done.